ledger-core/virtual/integration/delegationtoken_test.go

```diff
@@ -346,7 +347,7 @@ func TestDelegationToken_IsMessageFromVirtualLegitimate(t *testing.T) {
 						QueryRoleMock.Return([]reference.Global{fixedVe}, nil)
 				}
 
-				switch testCase.currentVE {
+				switch testCase.approverVE {
```

Comment:
because
```
func (s service) GetCallDelegationToken(outgoing reference.Global, to reference.Global, pn pulse.Number, object reference.Global) payload.CallDelegationToken {
	return payload.CallDelegationToken{
		TokenTypeAndFlags: payload.DelegationTokenTypeCall,
		Approver:          s.affinity.Me(),
```